setup.py

```diff
@@ -3,9 +3,9 @@
     name='consensus',
     version='1.0',
```

Comment:
Check if non-release version perhaps should have a different version while in master? (f.e. `-SNAPSHOT`, though not sure if Python guidelines has a different suggestion for this)

Reply:
The python guidelines are [not compatible](https://www.python.org/dev/peps/pep-0440/#semantic-versioning) with semantic versioning using hyphens. So we can:
- Just use `-SNAPSHOT` for now, stick to semantic versioning (as done with our Java code) and perhaps at a later moment investigate how we would want to use versioning.
- Decide to use PEP-440 guidelines for versioning right away, though needing more time to evaluate the versioning scheme.

As it is not a [final release](https://www.python.org/dev/peps/pep-0440/#final-releases), I'd still suggest to change it to something else.

Reply:
We don't have a real release pipeline ATM. We need to do that, but I think it's something to address when we pick up that story. The versionnumber is however incorrect at the moment, since the last release was 1.1. I'll update it to 2.0 because this PR contains breaking changes.

Reply:
Reason for the suggestion is so that you only have a single commit that has a "release version number" (which we could give a tag, even though we don't have a proper pipeline yet). But fair, we can then with that story also look at how such versioning should look like (semantic vs PEP-440). Good idea to increase the version number though!